code/datums/diseases/blackcold.dm

```diff
@@ -0,0 +1,67 @@
+/datum/disease/blackdeath
+	name = "The Black Death"
+	max_stages = 3
+	spread = "Airborne"
+	cure = "Spaceacillin"
+	cure_id = "spaceacillin"
+	cure_chance = 10
+	agent = "Bl3ck V3tosi"
+	affected_species = list("Human", "Monkey")
+	permeability_mod = 0.75
+	desc = "If left untreated the subject will feel a insane fever and slowly succumb into a coma. Caused by eating mice" //Mice shaming is a good thing --Drago
```

Comment:
Severe fever, not insane. One slips into a coma and doesn't succumb to it. Needs a period at the end.